modules/database/src/main/java/com/intuit/wasabi/database/impl/DBITransaction.java

```diff
@@ -41,9 +42,8 @@
 public class DBITransaction implements Transaction {
 
     private static final Logger LOGGER = getLogger(DBITransaction.class);
-    private Handle handle;
     private DBI dbi;
-    private boolean inTransaction = false;
+    private AtomicBoolean inTransaction = new AtomicBoolean(false);
```

Comment:
Also, inTransaction is being just set and the value not checked anywhere. Neither are there any getters for it so that it could be used outside the scope of this class. It makes it redundant. Can we just remove it.